docs/pages/access-controls/compliance-frameworks/fedramp.mdx

```diff
@@ -13,32 +13,63 @@ Teleport FIPS mode works and how it can help your company to become FedRAMP auth
 
-## Download and install
+### Configuration Management
 
-Teleport Enterprise customers can download the custom FIPS package from their
-[Teleport account](https://teleport.sh). Look for `Linux 64-bit (FedRAMP/FIPS)`.
+| Control | Teleport Features |
+| - | - |
+| [CM-08 Information System Component Inventory]((=fedramp.control_url=)CM-08) | Teleport maintains a live list of all nodes within a cluster. This node list can be queried by users (who see a subset they have access to) and administrators any time. |
+
+### Identification and Authentication
 
+| Control | Teleport Features |
+| - | - |
+| [IA-02 Concurrent Session Control]((=fedramp.control_url=)IA-02) | Integrates with SSO providers such as GitHub, Okta, Google, etc. Acts as its own SSO provider. Enforces the use of multi-factor authentication (MFA), including requiring per-session MFA. Supports PIV-compatible hardware keys. |
+| [IA-04 Identifier Management]((=fedramp.control_url=)IA-04) | Maintains several unique identifiers: local users are required to be unique (unique username), roles have unique names and tied to organization roles via SSO, identifiers for devices are unique randomly generated IDs (UUID). |
+| [IA-08 Identification and Authentication (Non-Organizational Users)]((=fedramp.control_url=)IA-08) | Teleport supports PIV-compatible hardware keys. |
+| [IA-03 Device Identification and Authentication]((=fedramp.control_url=)IA-03) | Teleport requires valid x509 or SSH certificates issued by a Teleport Certificate Authority (CA) to establish a network connection for device-to-device network connection between Teleport components. |
+
+### System and Communications Protection
+
+| Control | Teleport Features |
+| - | - |
+| [SC-10 Network Disconnection]((=fedramp.control_url=)SC-10) | Teleport requires valid X.509 or SSH certificates issued by a Teleport Certificate Authority (CA) to establish a network connection for device-to-device network connection between Teleport components. |
+| [SC-12 Cryptographic Key Establish and Management]((=fedramp.control_url=)SC-12) | Teleport initializes cryptographic keys that act as a Certificate Authority (CA) to further issue x509 and SSH certificates. SSH and x509 user certificates that are issued are signed by the CA and are (by default) short-lived. SSH host certificates are also signed by the CA and rotated automatically (a manual force rotation can also be performed).<br/>Teleport Enterprise builds against a FIPS 140-2 compliant library (BoringCrypto) is available. <br/>In addition, when Teleport Enterprise is in FedRAMP/FIPS 140-2 mode, Teleport will only start and use FIPS 140-2 compliant cryptography. |
```

Comment:
I don't think SSH host certificates are rotated automatically.

Should we mention HSM support here too?

Reply:
HSM isn't part of SC-12 but it seems like a good thing to mention here. For the automatic cert rotation note, the same line is also in SC-17. From what I've read, FedRAMP doesn't mandate any sort of automatic certificate rotation, but from this it looks like we do support it, unless I am misunderstanding (under rotation types):
https://goteleport.com/docs/management/operations/ca-rotation/

> Semi-automatic: Teleport automatically transitions between phases of the rotation after some amount of time (known as a grace period) elapses.

(Note, if it does need to be removed we will need to update the source webpage as well: https://goteleport.com/use-cases/fedramp-compliance).

Reply:
- Added HSM sentence. 
- Deleted rotation lines as discussed.